lib/application.js

```diff
@@ -58,7 +58,10 @@ class BookkeepingApplication {
             if (monalisaConfig.enableSynchronization) {
                 const monALISASynchronizer = await this.createMonALISASynchronizer();
                 this.scheduledProcessesManager.schedule(
-                    () => monALISASynchronizer.synchronizeDataPassesFromMonALISA(),
+                    async () => {
+                        await monALISASynchronizer.synchronizeDataPassesFromMonALISA();
+                        await monALISASynchronizer.synchronizeSimulationPassesFromMonALISA();
```

Comment:
Maybe you should run both in parallel?

Reply:
no, it must go sequentially, because simulation passes are dependent on data passes in DB